### DISCUSSION TITLE
O que acham deste curso?

### DISCUSSION THREAD

**Discussion by julia-droid** [frontendbr/forum#1603] [Discussão]
Pessoal, como já postei aqui, estou estudando bastante e seguindo as dicas de vocês, tanto pela documentação quanto por conteúdos no youtube e blogs.

Vou também investir em um curso, e vi que esta semana o Dev Pleno lançou o curso Fullstack, conhecem? Algém já fez? recomendam?

Esse curso: http://bit.ly/fullstack-dev-pleno

Vi que aborda de maneira bem fácil para iniciantes mesmo, e promete ir aos poucos até o avançado avançado.

Acompanhei vários conteúdos deles e achei bem legal e fácil de entender e evoluir... As vezes pego conteúdos que acho muito avançado e não entendo nada rs

Acham que vale a pena?

**Comment by wilsonneto-swe** [comment]
Em se tratando de cursos com a stack js eu fiz os cursos e ainda me considero aluno do Dev Pleno e da Rocketseat, a galera é muito foda e todos os cursos. 

Tem muitos outros cursos bem legais, mas estes dois são qualidade garantida, me ajudaram bastante.

São projetos práticos, eu junto isso com muita leitura da documentação e fazer meus projetos, ai fechou, para mim a melhor maneira de aprender.

Só acho que seria legal primeiro focar em apenas uma tecnologia, eu focaria no curso do Dev Pleno de React que gostei bastrante, para depois o fullstack.

Veja o curso de react também:
http://bit.ly/reactjs-dev-pleno


-----  
By: :relaxed: Wilson Neto  
[Linkedin](https://linkedin.com/in/wilsonnetobr/)
[Blog](http://wilsonneto.com.br)
[Github](https://github.com/wilsonneto-dev)
